financials/Funding request template.txt

```diff
@@ -0,0 +1,24 @@
+## Template for funding request
+This funding request template was made based on past experiences and debates on the Grin Forum about the necessity to have clear, 
+transparent and unambiguous information in funding requests. 
+This template is not intended to be exhaustive and only serves as a guideline to clearly communicate the minimal amount of information 
+needed for the GRIN community to assess requests for funding. Feel free to add and modify to this template to improve clarity and transparency. 
+However, let us aim to keep this funding template minimal and elegant like GRIN and the Mimble Wimble protocol itself. 
+For ease of comparison of funding requests, it is advised to provide these fields of information in the order given below.
+
+## Financial information
+�	Funding period: 
+o	(e.g. 3 months)
+�	Total amount of requested funding:
+o	State total amount of requested funding in this period as well as in the preceding funding period both in USD and local currency
+�	Requested funding per month and number FTE�s
+o	E.g. 1 FTE is equivalent to 40 hours per week

```

Comment:
1. It's not a given that a funding request is monthly. Somebody might be estimating work in weeks, and have a good reason for that. Or hours, if it's a security audit for example. It might make sense to keep this open, but request that the unit is expressed clearly. 
1. This is the first time I hear about this `FTE unit`. Is it possible to do something simpler, i.e. indicate whether it's full-time (40hrs) or part-time, optionally assessing how many hours?